pkg/registry/tags_test.go

```diff
@@ -235,7 +198,45 @@ func TestTagsSort(t *testing.T) {
 
 	for _, tt := range testCases {
 		tt := tt
+		repotags := []string{
+			"0.1.0",
+			"0.4.0",
+			"3.0.0-beta.1",
+			"3.0.0-beta.3",
+			"3.0.0-beta.4",
+			"4",
+			"4.0.0",
+			"4.0.0-beta.1",
+			"4.1.0",
+			"4.1.1",
+			"4.10.0",
+			"4.11.0",
+			"4.12.0",
+			"4.13.0",
+			"4.14.0",
+			"4.19.0",
+			"4.2.0",
+			"4.20",
+			"4.20.0",
+			"4.20.1",
+			"4.21",
+			"4.21.0",
+			"4.3.0",
+			"4.3.1",
+			"4.4.0",
+			"4.6.1",
+			"4.7.0",
+			"4.8.0",
+			"4.8.1",
+			"4.9.0",
+			"ubuntu-5.0",
+			"alpine-5.0",
+			"edge",
+			"latest",
+		}
```

Comment:
Can we keep this var on top as previous?

Reply:
So despite the fact that `SortTags` returns a slice, it actually sorts the slice in place and then returns it again.

This is an issue when running in parallel since the jobs sort simultaneously the same slice resulting in incorrect results.

The remedies would be to make a copy of the slice before sorting. This could either be done in the test function or within `SortTags` so we sort a copy of the input.

I guess if we decide to do make the copy in the test function, to make it more clear that it sorts in place we should probably remove the return value so that `SortTags` works more like `strings.Sort` so it's clear it's doing a sort in place.

Reply:
Indeed, I didn't take that into account, sounds good then!